src/mlpack/core/cv/metrics/topk_accuracy_impl.hpp

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file topk_accuracy_impl.hpp
+ * @author Arunav Shandeelya
+ * 
+ * The implementation of class topK_accuracy_score
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+
+namespace mlpack {
+namespace cv {
+    template<typename MLAlgorithm, typename DataType, typename ResponseType, typename TopK>
+    double TopK_Accuracy::Evaluate(MLAlgorithm& model,
+                                    const DataType& data,
+                                    const arma::Row<size_t>& labels,
+                                    const TopK& k)
```

Comment:
Could you also fix the spacing here. Thanks.

Reply:
yeah sure. fixed